web/src/compose_actions.js

```diff
@@ -336,6 +336,16 @@ export function cancel() {
     $(document).trigger("compose_canceled.zulip");
 }
 
+export function on_view_change() {
```

Comment:
Let's call this `on_show_navigation_view` -- I don't want any ambiguity about whether a narrow qualifies.